src/storage/v2/indices/point_index.cpp

```diff
@@ -67,7 +67,11 @@ bool PointIndexStorage::CreatePointIndex(LabelId label, PropertyId property,
   for (auto const &v : vertices) {
     if (v.deleted) continue;
     if (!utils::Contains(v.labels, label)) continue;
-    auto value = v.properties.GetProperty(property);
+
+    auto maybe_value = v.properties.GetPropertyOfTypes(property, std::array{PropertyStoreType::POINT});
```

Comment:
maybe lifetime extension problems...extract to its own static constexpr variable